proof/crefine/ARM/Tcb_C.thy

```diff
@@ -2611,21 +2598,20 @@ lemma slotCapLongRunningDelete_ccorres:
        apply (clarsimp simp: typ_heap_simps cap_get_tag_isCap
                              from_bool_0
                       dest!: ccte_relation_ccap_relation)
-       apply (simp add: from_bool_def false_def true_def
+       apply (simp add: from_bool_def
                  split: bool.split)
        apply (auto simp add: longRunningDelete_def isCap_simps
                  split: capability.split)[1]
       apply simp
       apply (wp hoare_drop_imps isFinalCapability_inv)
      apply (clarsimp simp: Collect_const_mem guard_is_UNIV_def)
      apply (rename_tac rv')
-     apply (case_tac rv'; clarsimp simp: false_def true_def)
+     apply (case_tac rv'; clarsimp simp: false_def)
```

Comment:
This `false_def` wasn't able to be trivially removed. There are some rules that are more specific than I would like that would do it, but if anyone has any better ideas then the goal is
```
SCAST(32 signed → 32) false = ret__unsigned_long_' s ⟹ ret__unsigned_long_' s = 0
```

Reply:
This one looks fine to unfold false_def